website/docs/r/compute_project_metadata.html.markdown

```diff
@@ -8,7 +8,10 @@ description: |-
 
 # google\_compute\_project\_metadata
 
-Manages metadata common to all instances for a project in GCE.
+Manages metadata common to all instances for a project in GCE. If you
```

Comment:
What if we state that this resource authoritatively manages all of the project's metadata before we explain `google_compute_project_metadata_item` as an alternative? That way it will be more clear what the potential gotchas with using this resource are, and how the alternative solves them.